src/components/dashboard/Dashboard.js

```diff
@@ -832,7 +833,10 @@ const Dashboard = props => {
   const handleScroll = useCallback(
     ({ ...args }) => {
       dispatchScrollEvent()
-      handleScrollEnd(args)
+
+      if (isWeb) {
```

Comment:
I dont understand this. does it mean that on native we will not change the header size?

Reply:
HandleMomentumScroll is handling the calling for handles rollend.

So it still minimizes, but only after the scrolling has ended (at the same point that we show the toTop button)

Reply:
This is not a good solution. Most users on mobile simply scroll and do not stay on the exact position where it jitters. So now for most mobile users the UX experience will be worse.
The solution is either to make it less sensitive when it starts the animation or to not toggle the state until the previous animation is over. so once an animation starts (large->small or small->large) you set a reference to true. once the animation ends you set it to false. at any point if the reference is true we do not change the state of the header.

Reply:
so insted of if(isWeb) you would have if(isAnimationInProgress === false)

Reply:
I don't understand your concern with the solution.
On momentumScroll is triggered whenever the scrollbar stops scrolling.
The moment this happens, it minimizes the header.

This is the simplest approach out there.

(If it's scrolling and you touch feed again, it's also considered onMomentumScrolEnd)

Reply:
Also the issue is not in the animation running.

It's about an infinite loop where it does not know which animation to run because the condition changes every 1 ms